app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -360,6 +360,8 @@ sed -e 's/DEFAULTKERNEL=kernel-uek/DEFAULTKERNEL=kernel/g' -i /etc/sysconfig/ker
 
 <%= snippet 'insights' if host_param_true?('host_registration_insights') && os_major < 9 -%>
 
+<%= snippet 'disk_enc_clevis_tang' %>
```

Comment:
I think, it would be better to only load this snippet if disc enc should be used. So something you did below 'unless tang_server_list.blank? %>' 

Use '<%= -%>" (the minus before %>) which prevents adding useless empty lines.